SketchUp/Sketchup/View.rb

```diff
@@ -641,13 +631,13 @@ def line_width=(width)
   #   view = view.lock_inference inputpoint
   #   view = view.lock_inference inputpoint1, inputpoint2
   #
-  # @param [optional] inputpoint
+  # @param inputpoint [Sketchup::InputPoint] (optional)
```

Comment:
Change signature to `def lock_inference(inputpoint = nil, inputpoint2 = nil)` and let YARD consistently mark up the optional params. (Also help IDEs)